.circleci/config.yml

```diff
@@ -0,0 +1,43 @@
+version: 2
+
+jobs:
+  build:
+    docker:
+      - image: mbgl/ci:r4-linux-clang-3.9
+    working_directory: ~/mapbox-gl-js
+    steps:
+      - checkout
+      - restore_cache:
+          key: v1-mapbox-gl-js-cache
+      - run:
+          name: install dependencies
+          command: |
+            bash ./.circleci/dependencies.sh
```

Comment:
Let's move the `yarn install` part of this to its own `run` step, to measure the time independently.